docs/serverless/alerts/alert-suppression.mdx

```diff
@@ -12,7 +12,7 @@ status: in review
 <DocCallOut color="warning" title="Requirements and notice">
     - ((ml-cap)) rules have <DocLink slug="/serverless/security/ml-requirements">additional requirements</DocLink> for alert suppression.
 
-    - Alert suppression is in technical preview for threshold, indicator match, event correlation, and new terms rules. The functionality may be changed or removed in a future release. Elastic will work to fix any issues, but features in technical preview are not subject to the support SLA of official GA features.
+    - Alert suppression is in technical preview for event correlation rules. The functionality may be changed or removed in a future release. Elastic will work to fix any issues, but features in technical preview are not subject to the support SLA of official GA features.
```

Comment:
@approksiu Do we specify a tier here? Platinum in ESS would be complete tier?

Reply:
Kseniia asked me to follow up - looks like it's available in Essentials, so I would say no need to specify a tier?